pkgs/intreehooks/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, pytoml
+, pytest
+}:
+
+buildPythonPackage rec {
+  pname = "intreehooks";
```

Comment:
No need to add `intreehooks` to the overlay, I've backported it to 19.09.

Reply:
maybe it's worth keeping it?

>  attribute 'intreehooks' missing, at /builds/nix-community/poetry2nix/default.nix:209:34
(from CI)

Reply:
re-added the commit